internal/controller/services/monitoring/resources/opentelemetry-collector.tmpl.yaml

```
@@ -0,0 +1,86 @@
+    exporters:
+      prometheus:
+        endpoint: 0.0.0.0:8889
+        resource_to_telemetry_conversion:
+          enabled: true # by default resource attributes are dropped
+      {{ if .Traces }}
+      otlp:
+        endpoint: "tempo-<example>-distributor:4317" 
+        tls:
+          insecure: true
+      {{ end }}
+    service: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)